### ISSUE TITLE
is there a demo page?

### ISSUE THREAD

**Issue by yairEO** [dy/connection-line#1]
I would very much like to see demo page, codepen or even a gif showing this in action before I'll spend time downloading this and configuring on my project...

`test.html` file doesn't even work..

makes sense no?

**Comment by dy** [comment]
Yeh, makes total sense. Though that is experimental, I will try to give that a shot one day.

**Comment by ueberhammDesign** [comment]
+1

**Comment by foobarbecue** [comment]
Yeah... I spent some time trying to use this but no luck. Without a working example I have to just assume it's broken.

**Comment by nicooprat** [comment]
Got it working with:

```css
connection-line {
  position: absolute;
  pointer-events: none;
}
```

```js
const connector = new Connector({
  from: connectors[i],
  to: connectors[i+1]
})
document.body.appendChild(connector.element)
connector.update() // This one is important!
```

Do you plan on spending some time to finish this lib? :) Looks like it's exactly what I need! Thanks.

**Comment by dy** [comment]
Yeah, def! Just really out of time for now, but going to give it a time soon as a part of audio-lab/lab
